src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -156,50 +160,84 @@ class ArchiveExplorer extends React.Component {
         return itemCollection.filter(item => fuzzySearch(trimmedQuery, item.name, 0));
     };
 
+    /**
+     * Handle sort click
+     *
+     * @param {object} sort
```

Comment:
Do we still need this parameter if we do the destruction? It's not used anywhere in the function body.

Reply:
Yes, this will help with type hinting for browsers that support it.